sympy/stats/drv_types.py

```diff
@@ -834,3 +838,85 @@ def Zeta(name, s):
 
     """
     return rv(name, ZetaDistribution, s)
+
+
+#-------------------------------------------------------------------------------
+# Delaporte distribution ------------------------------------------------------------
+
+class DelaporteDistribution(SingleDiscreteDistribution):
+    _argnames = ('lamda', 'alpha', 'beta')
+    set = S.Naturals0
+
+    @staticmethod
+    def check(lamda, alpha, beta):
+        _value_check(lamda > 0, 'lambda should be positive')
+        _value_check(alpha > 0, 'alpha should be positive')
+        _value_check(beta > 0, 'beta should be positive')
+
+    def pdf(self, k):
+        lamda = self.lamda
+        alpha = self.alpha
+        beta = self.beta
+        i = Symbol('i', integer=True)
```

Comment:
Is it possible that the user used symbol `i` as one of the parameters? If so, there is a collision here.

Reply:
Yeah, seems like there is collision but how can we avoid it? Should we mention in the docstring explicity?

Reply:
May be you could use `i = Dummy('i', integer=True)`